test/client_add_user_properties_test.rb

```diff
@@ -53,6 +64,17 @@ def test_add_user_properties_with_long_string_property_value
         'long; 1025 is above the 1024-character limit', exception.message
   end
 
+  def test_add_user_properties_with_long_identity
```

Comment:
This should be `test_add_user_properties_with_long_identity_symbol`. Right now, I think you're redefining the method on line 34.

Also, can you please move this test right below the identity length-related test?